src/App.js

```diff
@@ -1,7 +1,37 @@
-import "./App.css";
+import React from "react";
+import ItemList from "./Component/ItemList";
+import CheckOut from "./Component/CheckOut";
+import Cart from "./Component/Cart"
 
-const App = () => {
-  return <h1>Hello, world!</h1>;
-};
 
-export default App;
+
+class App extends React.Component {
+   state ={cart:[], subtotal:0}
+
+
+ addCart = (item) => {
+   this.setState((prevState) => {
+    this.subtotal(item.price)
+     return {cart:[item, ...prevState.cart]}
+   })
+ }
+ subtotal = (tot) => {
+   this.setState((prevState)=>{
+     return {subtotal: tot + prevState.subtotal}
+   })
+   
+ }
+  render() {
+    console.log(this.state.subtotal)
```

Comment:
This is where you can calculate subtotal, tax, and total. You should also pass the total price to CheckOut as a prop because CheckOut needs to alert the user with the total price when the form is submitted.